metadata.yaml

```diff
@@ -3,7 +3,7 @@
 # between patch versions.
 #
 # update this file only when a new major or minor version is released
-apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha4
```

Comment:
clusterctl is still at `v1alpha3` - I don't think this should be updated.

Reply:
c/f 
https://github.com/kubernetes-sigs/cluster-api/blob/de4b6521117ad5a473548c3388112d8af8209566/metadata.yaml#L6

Reply:
lol, i thought i was being overly smart here. thanks!